stable/kibana/README.md

```diff
@@ -40,6 +40,7 @@ Parameter | Description | Default
 --- | --- | ---
 `affinity` | node/pod affinities | None
 `env` | Environment variables to configure Kibana | `{}`
+`extraEnv` | `containers[].env` entries to configure Kibana | `{}`
```

Comment:
Can you add a bit more as to why a user would use extraEnv versus env? That would be helpful.